docs/data-sources/volumes.md

```diff
@@ -0,0 +1,39 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_volumes Data Source
+
+Retrieves a list of [databricks_volume](../resources/volume.md) ids (full names), that were created by Terraform or manually.
+
+## Example Usage
+
+Listing all volumes in a _things_ [databricks_schema](../resources/schema.md) of a  _sandbox_ [databricks_catalog](../resources/catalog.md):
+
+```hcl
+data "databricks_volumes" "this" {
+  catalog_name = "sandbox"
+  schema_name = "things"
+}
+
+output "all_volumes" {
+  value = data.databricks_volumes.this
+}
+```
+
+## Argument Reference
+
+* `catalog_name` - (Required) Name of [databricks_catalog](../resources/catalog.md)
+* `schema_name` - (Required) Name of [databricks_schema](../resources/schema.md)
+
+## Attribute Reference
+
+This data source exports the following attributes:
+
+* `ids` - a set of [databricks_volume](../resources/volume.md) full names: *`catalog`.`schema`.`volume`*
```

Comment:
nit: Since we are mentioning above: `Retrieves a list of`, should we also mention list here instead of set for consistency?

Reply:
Changes made.